modules/ROOT/pages/install-rtfctl.adoc

```diff
@@ -179,6 +180,14 @@ rtfctl wait <value_in_seconds> --namespace <rtf_namespace>
 rtfctl wait --timeout <value_in_seconds> --namespace <rtf_namespace>
 ----
 
+=== `renew`
+
+Renews an expired client certificate using a pre-downloaded package
+
+----
+rtfctl renew-expired-client-cert <cert-bundle-filename> -namespace <rtf_namespace>
+----
```

Comment:
Maybe we don't need to put `-namespace <rtf_namespace>`
only `rtfctl renew-expired-client-cert <cert-bundle-filename>` Thanks.